services/client/src/pages/events/index.js

```diff
@@ -20,7 +21,8 @@ const mapStateToProps = function(state) {
     hasErrors: hasErrors(state),
     isLoading: isLoading(state),
     hasMoreItems: hasMoreItems(state),
-    page: page(state)
+    page: page(state),
+    location: location(state)
```

Comment:
`location` is plucked from the events state, however, we should avoid setting the subdomain as the default; instead, we should update `mapDispatchToProps` with an action and action creator to add the `location` to the events state for us. The action creator to set the subdomain should be fired and resolved before `fetchData` is fired.

https://github.com/apoclyps/my-dev-space/pull/503/files#diff-6d7687c45f3aedf399c51b25381c5cdcR31

Reply:
I made this intentionally because the default value (when there is not a subdomain) is just an empty string.